AppLovin MAX Demo App - Kotlin/app/src/main/java/com/applovin/enterprise/apps/demoapp/ui/BaseAdActivity.kt

```diff
@@ -36,9 +36,9 @@ abstract class BaseAdActivity : AppCompatActivity()
      * Log ad callbacks in the RecyclerView.
      * Uses the name of the function that calls this one in the log.
      */
-    protected fun logCallback()
+    protected fun logCallback(index: Int = 2)
```

Comment:
Doesn't this change the behavior for the other callbacks? The default used to be `1`

Reply:
After changed the param with a default value, there will be one more line in stacktrace called "use default", that makes all index should be increased by 1.